### PR TITLE
Add support long messages (UDH)

### PR DESCRIPTION
**Pull request by zaikin**
Very quick and dirty implementation.

### CODE REVIEW

File: smpp/transmitter.go [hunk 1]
```diff
@@ -253,6 +256,68 @@ func (t *Transmitter) Submit(sm *ShortMessage) (*ShortMessage, error) {
 	return sm, resp.Err
 }
 
+// SubmitLongMsg sends a long message (more than 140 bytes)
+// and returns and updates the given sm with the response status.
+// It returns the same sm object.
+func (t *Transmitter) SubmintLongMsg(sm *ShortMessage) (*ShortMessage, error) {
```
Comment (haisum) [inline smpp/transmitter.go]: Spelling mistake in Submit here

**Comment by AlekSi** [comment]
@fiorix Should we do anything else?

**Comment by fiorix** [comment]
Forgot about this, thanks for fixing it. :+1:

**Comment by AlekSi** [comment]
Are you ok with new method `SubmitLongMsg()`? I _think_ it can be merged with `Submit()`.

**Comment by fiorix** [comment]
I think the automatic approach is better, but I'd keep them in separate functions. For example, make `submitLongMsg()` not exported and put a check in Submit to call it when needed.

**Comment by fiorix** [comment]
On the response, we have to tell the caller how many parts were send, they might need this for billing.

**Comment by haisum** [comment]
`SubmitLongMsg(sm *ShortMessage) (*ShortMessage, error)` returns `*ShortMessage` which represents last sent message. Shouldn't we be returning array containing ShortMessage object of each sent message instead of just last one's.

**Pull request merged**